src/Samples/DIPS.Xamarin.UI.Samples/Controls/Toast/ToastPage.xaml.cs

```diff
@@ -0,0 +1,113 @@
+using System.Threading.Tasks;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Controls.Toast;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Color = System.Drawing.Color;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Toast
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ToastPage : ContentPage
+    {
+        private string m_pageTitle;
+        private ICommand m_showToastCommand;
+        private ICommand m_showToastCommand2;
+        private ICommand m_showToastCommand3;
+
+        public ToastPage()
+        {
+            InitializeComponent();
+
+            ConfigureToast();
+
+            PageTitle = "Hello, World!";
+
+            ShowToastCommand = new Command(parameter =>
+            {
+                Toaster.Current.ToastAction = async () =>
+                {
+                    PageTitle = "Hello, Mercury!";
+                    await Task.Delay(1500);
+                    PageTitle = "Hello, World!";
+                };
+                Toaster.Current.Text = parameter.ToString();
+
+                Toaster.Current.ShowToaster();
+            });
+
+            ShowToastCommand2 = new Command(parameter =>
+            {
+                Toaster.Current.ToastAction = null;
+                Toaster.Current.Text = parameter.ToString();
+
+                Toaster.Current.ShowToaster();
+            });
+
+            ShowToastCommand3 = new Command(parameter =>
+            {
+                Toaster.Current.ToastAction = async () =>
+                {
+                    await Toaster.Current.HideToaster();
+                };
+                Toaster.Current.Text = parameter.ToString();
+
+                Toaster.Current.ShowToaster();
+            });
+        }
+
+        public string PageTitle
+        {
+            get => m_pageTitle;
+            set
+            {
+                m_pageTitle = value;
+                OnPropertyChanged(nameof(PageTitle));
+            }
+        }
+
+        public ICommand ShowToastCommand
+        {
+            get => m_showToastCommand;
+            set
+            {
+                m_showToastCommand = value;
+                OnPropertyChanged(nameof(ShowToastCommand));
+            }
+        }
+
+        public ICommand ShowToastCommand2
+        {
+            get => m_showToastCommand2;
+            set
+            {
+                m_showToastCommand2 = value;
+                OnPropertyChanged(nameof(ShowToastCommand2));
+            }
+        }
+
+        public ICommand ShowToastCommand3
+        {
+            get => m_showToastCommand3;
+            set
+            {
+                m_showToastCommand3 = value;
+                OnPropertyChanged(nameof(ShowToastCommand3));
+            }
+        }
+
+        private static void ConfigureToast()
+        {
+            Toaster.Current.FontSize = 13;
```

Comment:
Multiple threads with different "plans" will fight for setting properties like these?

Reply:
To make sure the toast displays the message of the immediate action, isn't it much better to override the last toast?

Since the design has followed to replace the last toast, there won't be any conflicts in the properties.

Reply:
I tried to start a discussion about this I  the spec, but no reply.
https://github.com/DIPSAS/DIPS.Xamarin.UI/issues/196#issuecomment-652796923

Reply:
For this version at least, shall focus on having only one toast at a time?